src/spikeinterface/core/core_tools.py

```diff
@@ -934,3 +941,41 @@ def convert_bytes_to_str(byte_value: int) -> str:
         byte_value /= 1024
         i += 1
     return f"{byte_value:.2f} {suffixes[i]}"
+
+
+def spike_vector_to_dict(spike_vector: np.ndarray) -> dict:
```

Comment:
This returns a list doesn't?

Reply:
Also needs documentation.

Reply:
> This returns a list doesn't?

Correct!
Forgot to change that when I implemented multi-segments ^^'